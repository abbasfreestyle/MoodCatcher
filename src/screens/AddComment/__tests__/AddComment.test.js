import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { AddCommentScreen } from '..';

const requiredProps = {
  navigation: { navigate: jest.fn() },
  comment: 'Test comment',
  onUpdateComment: jest.fn()
};

const wrapper = shallow(<AddCommentScreen {...requiredProps} />);

describe('Screens > Add Comment', () => {
  describe('Snapshots', () => {
    test('should match', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
