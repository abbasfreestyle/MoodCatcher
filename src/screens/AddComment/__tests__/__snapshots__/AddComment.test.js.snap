// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens > Add Comment Snapshots should match 1`] = `
<View
  flex={1}
>
  <Title>
    Add a note
  </Title>
  <KeyboardAvoidingView
    behavior="padding"
    enabled={true}
    keyboardVerticalOffset={50}
    style={
      Object {
        "flex": 1,
        "padding": 20,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      multiline={true}
      onChangeText={[Function]}
      placeholder="Help remind yourself why you feel this way... (optional)"
      rejectResponderTermination={true}
      underlineColorAndroid="transparent"
      value="Test comment"
    />
  </KeyboardAvoidingView>
  <KeyboardAvoidingView
    behavior="position"
    enabled={true}
    keyboardVerticalOffset={90}
  >
    <Regular
      disabled={false}
      flex={true}
      margin={10}
      onPress={[Function]}
    >
      Submit Mood
    </Regular>
  </KeyboardAvoidingView>
</View>
`;
